content/en/docs/tasks/traffic-management/egress/egress-tls-origination/index.md

```diff
@@ -203,8 +203,284 @@ topics and articles but does not prevent an attackers from learning that `editio
     $ kubectl delete destinationrule edition-cnn-com
     {{< /text >}}
 
-1.  Shutdown the [sleep]({{< github_tree >}}/samples/sleep) service:
+## Mutual TLS origination for egress traffic
+
+Similar to the previous section, this section describes how to configure a sidecar to perform
```

Comment:
Seems like a lot of duplicate material from the egress gateway task. I would suggest trying to get the common part into a single section (either in this doc, or separate), and then change the other doc (or both) to just say something like: follow [these instruction](link) to set up an external mutual TLS server for the example.